packages/evm/src/evm.ts

```diff
@@ -150,7 +148,7 @@ export class EVM implements EVMInterface {
 
   public readonly _transientStorage: TransientStorage
 
-  public readonly events: AsyncEventEmitter<EVMEvents>
+  public readonly events: AsyncEventEmitter
```

Comment:
Ugh.

@jochem-brouwer Seems we are loosing this typing here. 😬

I've tried to re-add, but seems evenemitter2 is not generic so this is not an option.

Hmm, sorry for that, I know this was some great improvement. Since we were so stuck on this import problem finally solved here and having this super-old dependency removed and updated would generally be really good I would have some stronger tendency for now to nevertheless go for it. Are you ok with this? 😬

I am currently not seeing an alternative.

Or are there eventual solutions here to get the typing back? Can't think of any right now?

Reply:
Seems this is being worked on (respectively: already done) here? 🤩

https://github.com/EventEmitter2/EventEmitter2/pull/294

So maybe we can just bring this back in a follow-up bugfix release once this is ready to be published?